tests/utils/helpers.rs

```diff
@@ -835,6 +837,14 @@ impl TestWallet {
             .collect()
     }
 
+    pub fn fungible_history(
+        &self,
+        contract_id: ContractId,
+        iface: impl Into<IfaceRef>,
+    ) -> Result<HashMap<XWitnessId, IfaceOp<AmountChange>>, WalletError> {
```

Comment:
please change the return output to `HashMap<XWitnessId, IfaceOp<AmountChange>>` (hence call `unwrap` after call to `fungible_history`)